cmd/controller/main.go

```diff
@@ -43,6 +47,20 @@ func main() {
 	lo.Must0(op.AddHealthzCheck("cloud-provider", awsCloudProvider.LivenessProbe))
 	cloudProvider := metrics.Decorate(awsCloudProvider)
 
+	client := op.Manager.GetAPIReader()
+	nodeClassList := v1beta1.EC2NodeClassList{}
+	err := client.List(ctx, &nodeClassList)
+	if err != nil {
+		logging.FromContext(ctx).Fatalf("failed to list EC2NodeClasses on startup, %s", err.Error())
```

Comment:
```suggestion
		logging.FromContext(ctx).Fatalf("listing EC2NodeClasses on startup, %s", err.Error())
```

nit: consider using the gerund since we use this everywhere for wrapping errors in the codebase